openwisp_controller/config/apps.py

```diff
@@ -172,6 +175,21 @@ def enable_cache_invalidation(self):
             DeviceChecksumView.invalidate_checksum_cache,
             dispatch_uid='invalidate_checksum_cache',
         )
+        post_save.connect(
+            devicegroup_change_handler,
+            sender=self.devicegroup_model,
+            dispatch_uid='invalidate_devicegroup_cache_on_devicegroup_change',
+        )
+        pre_save.connect(
```

Comment:
I think it would be more useful to have post_delete of cert rather than the post save of the org

Reply:
I don't understand this. 😅